src/arm/BBORG_SERVO-00A2.dts

```diff
@@ -35,27 +35,27 @@
 
     P9_19_pinmux { status = "disabled"; };  /* i2c2.scl */
     P9_20_pinmux { status = "disabled"; };	/* i2c2.sda */
-    P8_10_pinmux { status = "disabled"; };	/* servo.enable */
+    P8_10_pinmux { status = "disabled"; };	/* pca9685.enable */
 };
 
 &am33xx_pinmux {
 
     bborg_servo_input_pins: pinmux_bborg_servo_input_pins {
         pinctrl-single,pins = <
-            BONE_P8_12 (PIN_OUTPUT_PULLDOWN | MUX_MODE7)    /* P8_12: gpio1_12 */
-            BONE_P8_11 (PIN_OUTPUT_PULLDOWN | MUX_MODE7)    /* P8_11: gpio1_13 */
-            BONE_P9_30 (PIN_OUTPUT_PULLDOWN | MUX_MODE7)    /* P9_30: gpio3_16 */
-            BONE_P9_27 (PIN_OUTPUT_PULLDOWN | MUX_MODE7)    /* P9_27: gpio3_19 */
-            BONE_P9_41 (PIN_OUTPUT_PULLDOWN | MUX_MODE7)    /* P9_41: gpio3_20 */
-            BONE_P9_42 (PIN_OUTPUT_PULLDOWN | MUX_MODE7)    /* P9_42: gpio0_7 */
+            BONE_P8_12 (PIN_INPUT_PULLDOWN | MUX_MODE7)    /* P8_12: gpio1_12 */
+            BONE_P8_11 (PIN_INPUT_PULLDOWN | MUX_MODE7)    /* P8_11: gpio1_13 */
+            BONE_P9_30 (PIN_INPUT_PULLDOWN | MUX_MODE7)    /* P9_30: gpio3_16 */
+            BONE_P9_27 (PIN_INPUT_PULLDOWN | MUX_MODE7)    /* P9_27: gpio3_19 */
+            BONE_P9_41 (PIN_INPUT_PULLDOWN | MUX_MODE7)    /* P9_41: gpio3_20 */
+            BONE_P9_42 (PIN_INPUT_PULLDOWN | MUX_MODE7)    /* P9_42: gpio0_7 */
         >;
     };
 
 
-    bborg_servo_pca9685_i2c_pins: bborg_servo_pca9685_i2c_pins {
+    bborg_servo_pca9685_i2c_pins: pinmux_bborg_servo_pca9685_i2c_pins {
```

Comment:
Why does this bus need to be reconfigured? Cant the default configuration be used?

Reply:
Yes, we can use the default I2C bus configuration. But, this will make sure the person using the cape sees the pins attached to the cape when he/she uses show-pins utility. There may be other benefits also :)

Reply:
I guess, but note that some of these capes are stackable. I2C2 is a shared resource for all capes and only a handful need to reconfigure those pins to be anything but the default I2C function.

Reply:
You are absolutely right, I forgot to take that into account.

Reply:
Have you updated? I think you can add new commits to your pull request, even if it has been merged.